src/geometry.jl

```diff
@@ -1,59 +1,93 @@
 """
     Shape
 
-Abstract type for the shape of the object being modelled.
+Abstract supertype for the shape of the organism being modelled.
 """
 abstract type Shape end
 
+"""
+    Insulation
+
+Abstract supertype for the shape of the organism being modelled.
```

Comment:
change to 'insulation of the organism being modelled'

Reply:
```suggestion
Abstract supertype for the insulation of the organism being modelled.
```